test/omnibridge_nft/common.test.js

```diff
@@ -1692,6 +1708,22 @@ function runTests(accounts, isHome) {
             const event = await getEvents(contract, { event: 'TokensBridged' })
             expect(event.length).to.be.equal(2)
           })
+
+          it('should use default name, which can be reset later', async () => {
+            const data = deployAndHandleBridgedERC1155({ tokenIds: [1, 2], values: [1, 3], name: '', symbol: '' })
```

Comment:
could you add a test to verify the functionality on a token contract that has no `name` and `symbol` methods at all?

Reply:
Sure, added such unit test